internal/dinosaur/pkg/workers/clusters_mgr.go

```diff
@@ -343,7 +345,11 @@ func (c *ClusterManager) reconcileDeprovisioningCluster(cluster *api.Cluster) er
 
 		// if it is the only cluster left in that region, set it back to ready.
 		if siblingCluster == nil {
-			return c.ClusterService.UpdateStatus(*cluster, api.ClusterReady)
+			err := c.ClusterService.UpdateStatus(*cluster, api.ClusterReady)
+			if err != nil {
+				return fmt.Errorf("reconciling deprovisioning cluster: %w", err)
```

Comment:
This message should be added by the caller of this function, I think. I would suggest something like 
```suggestion
				return fmt.Errorf("updating status for cluster %s to %s: %w", cluster.clusterID, api.ClusterReady, err)
```